aiida_quantumespresso/parsers/parse_raw/cp.py

```diff
@@ -208,108 +240,120 @@ def parse_cp_xml_output(data):
 
     # CARD CONTROL
 
-    cardname = 'CONTROL'
+    cardname='CONTROL'
```

Comment:
There are a bunch of stylistic changes here. We are using automatic styling: you probably want to install the pre-commit hooks and run them, to fix the problems.

Space problems are automatically fixed by yapf.

Then, there might be a few more suggestions by prospector that you will need to address to ensure the tests pass, see error log e.g. here: https://github.com/aiidateam/aiida-quantumespresso/pull/455/checks?check_run_id=732410782

Reply:
This guide from aiida-core should apply more or less also here: https://github.com/aiidateam/aiida-core/wiki/Development-environment#pre-commit-hooks